### PR TITLE
Add KB section and organize by issue type.

### PR DESCRIPTION
**Pull request by stbof**
https://cockroachlabs.atlassian.net/browse/DOC-2411

The section that addresses the Jira is https://deploy-preview-13416--cockroachdb-docs.netlify.app/docs/dev/query-behavior-troubleshooting.html#select-statements-with-group-by-columns-are-slow.

### CODE REVIEW

**Review by florence-crl — CHANGES_REQUESTED** [review]
Would you be able to change this confusing sentence to bullet points?

> Suppose you have a slow query has a GROUP BY clause, uses an index that has a STORING clause, and some or all of the columns in the query's GROUP BY clause are part of the index's STORING clause and are not index key columns

TO: 

In this scenario,
* the slow query has a GROUP BY clause,
* the slow query uses an index that has a STORING clause, and
* some or all of the columns in the query’s GROUP BY clause are part of the index’s STORING clause and are not index key columns.

File: v22.1/query-behavior-troubleshooting.md [hunk 1]
```diff
@@ -54,105 +56,202 @@ You can look more closely at the behavior of a statement by visualizing a statem
...
 
 -  [Table scan best practices](performance-best-practices-overview.html#table-scan-best-practices)
 
 -  [Indexes best practices](schema-design-indexes.html#best-practices)
 
-## Query is always slow
+### `SELECT` statements with `GROUP BY` columns are slow
 
-If you have consistently slow queries in your cluster, use the [Statement Details](ui-statements-page.html#statement-details-page) page to drill down to an individual statement and [collect diagnostics](ui-statements-page.html#diagnostics) for the statement. A diagnostics bundle contains a record of transaction events across nodes for the SQL statement.
+Suppose you have a slow query has a `GROUP BY` clause, uses an index that has a `STORING` clause, and some or all of the columns in the query's `GROUP BY` clause are part of the index's `STORING` clause and are not index key columns. For example:
```
Comment (stbof) [inline v22.1/query-behavior-troubleshooting.md:91]: Suppose you have a slow query that

-  Has a `GROUP BY` clause.
-  Uses an index that has a `STORING` clause.
-  Where some or all of the columns in the query's `GROUP BY` clause are part of the index's `STORING` clause and are **not** index key columns. 

For example:

Comment (stbof) [inline v22.1/query-behavior-troubleshooting.md:91]:
```suggestion
Suppose you have a slow query that

-  Has a `GROUP BY` clause.
-  Uses an index that has a `STORING` clause.
-  Where some or all of the columns in the query's `GROUP BY` clause are part of the index's `STORING` clause and are **not** index key columns. 

For example:
```

**Comment by stbof** [comment]
> Would you be able to change this confusing sentence to bullet points?
> 
> > Suppose you have a slow query has a GROUP BY clause, uses an index that has a STORING clause, and some or all of the columns in the query's GROUP BY clause are part of the index's STORING clause and are not index key columns
> 
> TO:
> 
> In this scenario,
> 
> * the slow query has a GROUP BY clause,
> * the slow query uses an index that has a STORING clause, and
> * some or all of the columns in the query’s GROUP BY clause are part of the index’s STORING clause and are not index key columns.

Fixed.

**Review by florence-crl — APPROVED** [review]
LGTM

**Pull request merged**